packages/aws-cdk-lib/aws-ecs-patterns/test/fargate/queue-processing-fargate-service.test.ts

```diff
@@ -487,6 +487,98 @@ testDeprecated('test Fargate queue worker service construct - with optional prop
   });
 });
 
+testDeprecated('test Fargate queue worker service construct - with cpu scaling strategy disabled', () => {
+  // GIVEN
+  const stack = new cdk.Stack();
+  const vpc = new ec2.Vpc(stack, 'VPC');
+  const cluster = new ecs.Cluster(stack, 'Cluster', { vpc });
+  cluster.addAsgCapacityProvider(new AsgCapacityProvider(stack, 'DefaultAutoScalingGroupProvider', {
+    autoScalingGroup: new AutoScalingGroup(stack, 'DefaultAutoScalingGroup', {
+      vpc,
+      instanceType: new ec2.InstanceType('t2.micro'),
+      machineImage: MachineImage.latestAmazonLinux(),
+    }),
+  }));
+  const queue = new sqs.Queue(stack, 'fargate-test-queue', {
+    queueName: 'fargate-test-sqs-queue',
+  });
+
+  // WHEN
+  new ecsPatterns.QueueProcessingFargateService(stack, 'Service', {
+    cluster,
+    memoryLimitMiB: 512,
+    image: ecs.ContainerImage.fromRegistry('test'),
+    command: ['-c', '4', 'amazon.com'],
+    enableLogging: false,
+    environment: {
+      TEST_ENVIRONMENT_VARIABLE1: 'test environment variable 1 value',
+      TEST_ENVIRONMENT_VARIABLE2: 'test environment variable 2 value',
+    },
+    queue,
+    maxScalingCapacity: 5,
+    minHealthyPercent: 60,
+    maxHealthyPercent: 150,
+    serviceName: 'fargate-test-service',
+    family: 'fargate-task-family',
+    platformVersion: ecs.FargatePlatformVersion.VERSION1_4,
+    circuitBreaker: { rollback: true },
+    disableCpuBasedScaling: true,
+  });
+
+  // THEN - QueueWorker is of FARGATE launch type, an SQS queue is created and all optional properties are set.
+  Template.fromStack(stack).hasResourceProperties('AWS::ECS::Service', {
+    DeploymentConfiguration: {
+      MinimumHealthyPercent: 60,
+      MaximumPercent: 150,
+      DeploymentCircuitBreaker: {
+        Enable: true,
+        Rollback: true,
+      },
+    },
+    LaunchType: 'FARGATE',
+    ServiceName: 'fargate-test-service',
+    PlatformVersion: ecs.FargatePlatformVersion.VERSION1_4,
+    DeploymentController: {
+      Type: 'ECS',
+    },
+  });
+
+  Template.fromStack(stack).hasResourceProperties('AWS::SQS::Queue', { QueueName: 'fargate-test-sqs-queue' });
+
+  Template.fromStack(stack).hasResourceProperties('AWS::ECS::TaskDefinition', {
+    ContainerDefinitions: [
+      Match.objectLike({
+        Command: [
+          '-c',
+          '4',
+          'amazon.com',
+        ],
+        Environment: [
+          {
+            Name: 'TEST_ENVIRONMENT_VARIABLE1',
+            Value: 'test environment variable 1 value',
+          },
+          {
+            Name: 'TEST_ENVIRONMENT_VARIABLE2',
+            Value: 'test environment variable 2 value',
+          },
+          {
+            Name: 'QUEUE_NAME',
+            Value: {
+              'Fn::GetAtt': [
+                'fargatetestqueue28B43841',
+                'QueueName',
+              ],
+            },
+          },
+        ],
+        Image: 'test',
+      }),
+    ],
+    Family: 'fargate-task-family',
+  });
```

Comment:
```suggestion
  // THEN - No CPU target tracking policy is created
  Template.fromStack(stack).resourcePropertiesCountIs('AWS::ApplicationAutoScaling::ScalingPolicy', {
    PolicyType: 'TargetTrackingScaling',
  }, 0);
```
We should validate that the target tracking policy is missing.
Also, a test case should be added to verify that `targetUtilizationPercent` works.

Reply:
Thanks, added.